.github/ISSUE_TEMPLATE/prepare_release.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Prepare release
+about: Execute tasks for the creation and publishing of a new release
+title: 'Prepare release 0.0.0'
+labels: release
```

Comment:
I don't think this label exists, should we create it? didn't find an existing label that seemed fit

Reply:
@gabrielmer the labels across the waku-org are managed in the labels.yml `pm/.github/labels.yml`, I added the label here: https://github.com/waku-org/pm/commit/d73c241f8785e5572cbd3ac6e6b21368352dd633